src/setup-beam.js

```diff
@@ -342,14 +337,6 @@ async function get(url0, pageIdxs) {
   return ret
 }
 
-function prependToPath(what) {
```

Comment:
I learned, yesterday, that `.addPath` existed. It's a nice replacement for this.